quarto.qmd

```diff
@@ -180,7 +182,13 @@ Most importantly, while the visual editor displays your content with formatting,
 
 ### Exercises
 
-<!--# TO DO: Add exercises. -->
+1.  Re-create the document in @fig-visual-editor using the visual editor.
+2.  Using the visual editor, insert a code chunk using the Insert menu and then the insert anything tool.
+3.  Using the visual editor, figure out how to:
+    a.  Add a footnote.
+    b.  Add a horizontal rule.
+    c.  Add a block quote.
+4.  In the visual editor, go to Insert \> Citation and insert a citation to the paper titled [Welcome to the Tidyverse](https://joss.theoj.org/papers/10.21105/joss.01686) using its DOI (digital object identifier), which is 10.21105/joss.01686. Render the document and observe how the reference shows up in the document. What change do you observe in the YAML of your document?
```

Comment:
One last thing that illustrates a cool visual editor feature — create some content in a google or word doc, copy and paste it, and then look at it in source mode.

Reply:
Great idea, thanks! Will add to source editor section as an exercise.